sale_product_variant_attribute_tax/readme/CONTEXT.rst

```diff
@@ -0,0 +1,9 @@
+This module is somehow independent of sale_variant_configurator, but right now it's linked for not doing 2 modules.
```

Comment:
This text is not suitable to be put here as it is. You have to work around it a bit. Expressions like "The question is that" or "This module + the other one" are for you, not for being reflected in a README.

Also, the explanation "This module is somehow independent of sale_variant_configurator, but right now it's linked for not doing 2 modules." is more for ROADMAP, not for here (and also to be adapted).

Reply:
Could you please give me a suggestion for adding as an explanation?

Reply:
For CONTEXT.rst:

```
As the product taxes are defined at product template level, you can't have some variants with an specific tax.

These modules family covers the gap, adding the taxes at attribute level. This design decision has been taken for allowing to define the tax only once, as other options like having taxes at product variant level will force you to define the specific taxes for each of the variants that are affected.
```

For ROADMAP.rst:

```
- This module requires `sale_variant_configurator` as a way to avoid 2 glue modules, but it may be independent.
```
```